react/src/lib/List/examples/KitchenSink.js

```diff
@@ -3,15 +3,17 @@ import {
   ListDefault,
   ListListItemHeader,
   ListListItemSeparator,
+  ListWithEventBubbling,
 } from './index';
 
 export default class ListKitchenSink extends React.Component {
   render() {
     return (
       <React.Fragment>
-        <ListDefault />
-        <ListListItemHeader />
-        <ListListItemSeparator />
+          <ListDefault/>
```

Comment:
Can you check your tabing/spaces compared to the current library?

Reply:
Done.